lib/crystalball/map_generator/helpers/path_filter.rb

```diff
@@ -16,7 +16,12 @@ def initialize(root_path = Dir.pwd)
         # @return relatve paths inside root_path only
         def filter(paths)
           paths
-            .select { |file_name| file_name.start_with?(root_path) }
+            .select { |file_name|
+              if file_name.nil?
+                file_name = ""
+              end
+              file_name.start_with?(root_path)
+            }
```

Comment:
```suggestion
            .select { |file_name| file_name&.start_with?(root_path) }
```

`nil` checks can be avoided with the `&.` operator.

Unless the `nil -> ""` was for functional reasons (in which case ignore me!)  I'm assuming `nil` isn't supposed to match in the case of `root_path == ""`.